src/Ninja/G1CP/Content/Tests/test228.d

```diff
@@ -0,0 +1,44 @@
+/*
+ * #228 Gorn can't be asked about after talking to Lares
+ *
+ * There does not seem an easy way to test this fix programmatically, so this test relies on manual confirmation.
+ *
+ * Expected behavior: Gorn is findable after talking to Lares.
+ */
+func int G1CP_Test_228() {
+    const string TEMP_TOPIC_NAME = "G1CP Test 228"; // Has to be a unique name with absolute certainty
+    const string CH1_JoinNC = ""; CH1_JoinNC = G1CP_Testsuite_GetStringConst("CH1_JoinNC", 0);
+    var C_Npc npc; npc = G1CP_Testsuite_FindNpc("PC_Fighter");
+    var int funcId; funcId = G1CP_Testsuite_CheckDialogFunc("ORG_801_Lares_BringListAnteil_Info");
+    var int infoId; infoId = G1CP_Testsuite_CheckInfo("ORG_801_Lares_BringListBack");
```

Comment:
The dialog `ORG_801_Lares_BringListBack` is not required to have been told for calling the dialog info function `ORG_801_Lares_BringListAnteil_Info`. The info function does not reference that dialog.